### PR TITLE
Refugee Dialogue Project: Nunez family

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary
```SUMMARY: Content "Expands dialogue options for Pablo and Dana Nunez"```

#### Purpose of change
Addresses https://github.com/CleverRaven/Cataclysm-DDA/issues/30579 partially. These are two refugees whose dialogue is currently simply stubs and merits expansion.

#### Describe the solution
Expands their dialogue with backstory as well as some responses to things going on in the refugee center.

Adds the ability to attempt to recruit them to your own base or to the Tacoma Commune. These features are not complete: you'll need a clinic to get them to your own base, and I do not yet have the functionality available to move NPCs to new overmaps. Everything else is in place for the Tacoma portion.

Adds the ability to trade 8 cups of flour for a loaf of sourdough bread from Dana. 

Adds a quest to get Dana her original sourdough starter, with which she can then make unique bread with a higher Fun score.

#### Describe alternatives you've considered
I have no other quest ideas for them currently but am open to suggestions.

#### Additional context
It's possible that I'll add dialogue related to them getting pregnant again at some point many days_since_cataclysm, but since there are no babies in game I will probably hold off on that.

I have done some basic playtesting and it all seems to work well.

### CODE REVIEW

**Comment by Rail-Runner** [comment]
Perhaps player could be a bad person and decide to use that starter themself to bake "unique" bread themself, even if it'd result in quest failure? I think someone could decide to go that way.

**Comment by Zireael07** [comment]
Them getting the baby at some point in the future (even if it has no other effects in game) would be an awesome thing, just for pure uplifting value (see, life continues even after cataclysm!)

**Comment by faefux** [comment]
> Perhaps player could be a bad person and decide to use that starter to bake "unique" bread themself, even if it'd result in quest failure? I think someone could decide to go that way.

Maybe someone else in center or someone else that has come into contact with it want's her legendary Sourdough culture they have heard so much about.

(Nothing to do with this PR necessarily but we should be calling it a Culture and not a Starter in game, as far as i understand.)

**Comment by I-am-Erk** [comment]
Sourdough starter and sourdough culture are synonyms, "starter" is more commonly used according to my highly enthusiastic wife.

Dana's starter isn't usable when you get it without quite a bit of reviving. She might be willing to share, I haven't decided yet.

**Comment by faefux** [comment]
> Sourdough starter and sourdough culture are synonyms, "starter" is more commonly used according to my highly enthusiastic wife.

Ah, it's probably more common in the US,sorry. :)

This is only a teeny thing, but:
> ""dynamic_line": "Dana and I were evacuated early, because of her pregnancy. They took us to a concentration center, and then we got on a bus to come here. ....

Is 'Concentration Center' the right phrasing? It might be technically correct but i don't see that as what either civilians would be calling it because of the obvious connotations there.

**Comment by I-am-Erk** [comment]
Pablo chooses that wording pretty deliberately.  He does not have a very high opinion of how they've been treated thus far.

**Comment by faefux** [comment]
Understandable, Thanks for the clarification!

**Comment by Rail-Runner** [comment]
> Dana's starter isn't usable when you get it without quite a bit of reviving. She might be willing to share, I haven't decided yet.

Well, either that or maybe let player do the reviving with high enough skill if they do want to use it themself instead (or in case she dies for whatever reason).

**Comment by I-am-Erk** [comment]
The amount of json addition and hassle for what is essentially going to be a normal batch of sourdough starter if you don't have Dana's family recipe is not really worth it for the niche circumstance. I'm not going to let players produce the same special sourdough bread if they steal her starter.

File: data/json/npcs/refugee_center/surface_refugees/NPC_Dana_Nunez.json [hunk 1]
```diff
@@ -120,16 +155,209 @@
...
+  {
+    "type": "talk_topic",
+    "id": "TALK_REFUGEE_Dana_Background",
+    "dynamic_line": "I was working in a nice restaurant as a baker, until <the_cataclysm>.  I was pregnant, and it was getting close to time to go on mat leave; lots of time on my feet, getting up stupid-early to make pastries, you know.  Because of the pregnancy we were evacuated early and shipped out this way, but our bus was attacked in the chaos.  We managed to escape, but had to hike the rest of the way, with the Singhs and a few other refugees that didn't make it.  I lost the baby.  It wasn't a good time.",
+    "responses": [
+      { "text": "Have you done any baking lately?", "topic": "TALK_REFUGEE_Dana_Background_baking" },
+      { "text": "What were you saying before?", "topic": "TALK_NONE" },
+      { "text": "I'd better get going.  Bye, Dana.", "topic": "TALK_DONE" }
+    ]
+  },
```
Comment (Night-Pryanik) [inline data/json/npcs/refugee_center/surface_refugees/NPC_Dana_Nunez.json]: Maybe add some line to sympathize Dana's loss?

**Comment by I-am-Erk** [comment]
Switching this back to WIP because as Night points out, I forgot to CP in Dana's dialogue about losing the baby, and because I just thought of a much better workaround for being unable to set targeting on an item fetch quest

**Pull request merged**